packages/devextreme/js/__internal/ui/context_menu/m_context_menu.ts

```diff
@@ -630,15 +623,13 @@ class ContextMenu extends MenuBase {
   }
 
   _shouldHideOnOutsideClick(e) {
-    // @ts-expect-error
-    const { closeOnOutsideClick, hideOnOutsideClick } = this.option();
+    const hideOnOutsideClick = this.option('hideOnOutsideClick');
```

Comment:
I would prefer not to use WA to avoid ts-error. It is better to leave a comment // @ts-expect-error so that attention can be drawn to this issue in the future.
```suggestion
    // @ts-expect-error hideOnOutsideClick does not exist on type 'Properties'
    const { hideOnOutsideClick } = this.option();
```

Reply:
I'm not sure, I did it for consistency with other parts of the code. Maybe your outline is actually better